src/main/java/com/playmonumenta/networkrelay/RemotePlayerManagerAbstraction.java

```diff
@@ -296,6 +299,7 @@ protected boolean registerServer(String serverId) {
 	}
 
 	protected boolean unregisterServer(String serverId) {
+		boolean isRemote = getServerId().equals(serverId);
```

Comment:
I believe isRemote should be flipped here

Reply:
Fixed